src/tests/ftest/control/ms_failover.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 """
-(C) Copyright 2021 Intel Corporation.
+(C) Copyright 2021-2022 Intel Corporation.
```

Comment:
Spurious copyright update?

Reply:
Yeah, I had a merge conflict so it updated it.  I thought someone had addressed that.